src/components/views/rooms/VoiceRecordComposerTile.tsx

```diff
@@ -178,8 +227,8 @@ export default class VoiceRecordComposerTile extends React.PureComponent<IProps,
 
         // only other UI is the recording-in-progress UI
         return <div className="mx_VoiceMessagePrimaryContainer mx_VoiceRecordComposerTile_recording">
-            <LiveRecordingClock recorder={this.state.recorder} />
-            <LiveRecordingWaveform recorder={this.state.recorder} />
+            <Clock seconds={this.state.seconds} />
+            <Waveform relHeights={this.state.relHeights} />
```

Comment:
Can we restore the component structure here? It's fairly important for maintenance reasons (outside of this repo) to have the components be extremely independent.